src/docs/asciidoc/12-extension.adoc

```diff
@@ -9,6 +9,8 @@ The following properties can be configured:
 |`url`              |`Property<String>`  |`unix:///var/run/docker.sock` (Unix), `tcp://127.0.0.1:2375` (Windows)                  |The server URL to connect to via Docker's remote API.
 |`certPath`         |`DirectoryProperty` |Value of environment variable `DOCKER_CERT_PATH` if set                     |The path to certificates for communicating with https://docs.docker.com/articles/https/[Docker over SSL].
 |`apiVersion`       |`Property<String>`            |`null`                     |The https://docs.docker.com/develop/sdk/#view-the-api-reference[remote API version]. For most cases this can be left null.
+| `httpConnectionTimeout`   | `Property<Long>`|`null` | Determines the timeout until a new connection is fully established, 0 means infinite (if the HTTP transport is used)
```

Comment:
Can you add a dot at the end of the sentences similar to how we do it with other properties. Maybe we should also say what `null` means in terms of defaults.

Reply:
I'll add that dot + null one.